debs/jammy/.Jenkinsci

```diff
@@ -0,0 +1,66 @@
+pipeline {
+  agent  { label "master" }
+  parameters {
+  string(defaultValue: 'archivematica', description: '', name: 'PACKAGE')
+  string(defaultValue: 'qa/1.x', description: '', name: 'BRANCH')
+  string(defaultValue: '', description: '', name: 'GIT_REPO')
+  string(defaultValue: '0F4A4D31', description: 'For production packages, use production key', name: 'GPG_ID')
+  string(defaultValue: '1.8.0', description: '', name: 'VERSION')
+  string(defaultValue: '-beta1', description: '', name: 'RELEASE')
+  string(defaultValue: 'qa/1.x', description: '', name: 'PACKBUILD_BRANCH')
+  string(defaultValue: 'jenkinsci', description: '', name: 'REPOSITORY')
+  }
+
+stages {
+  stage('Get code') {
+    steps {
+     script {
+     currentBuild.displayName = "#${BUILD_NUMBER} ${PACKAGE}"
+     currentBuild.description = "Branch: ${BRANCH} <br> Version: ${VERSION} <br> Release: ${RELEASE} <br> Repository: ${REPOSITORY}"
+      }
+
+      git branch: '${PACKBUILD_BRANCH}', url: 'https://github.com/artefactual-labs/am-packbuild'
+    }
+  }
+  stage('Build package'){
+    steps {
+      sh '''
+        cd debs/bionic/${PACKAGE}/
+        make PACKAGE=$PACKAGE BRANCH=${BRANCH} VERSION=${VERSION} RELEASE=${RELEASE} GPG_ID=${GPG_ID}
+      '''
+    }
+  }
+  stage('Update repository'){
+    steps {
+      sh '''
+        cd debs/bionic/${PACKAGE}/
```

Comment:
Should  the `bionic`s in this file be updated to `jammy`?

Reply:
Jenkinsci files updated. I added a new variable called "DISTRO_CODENAME" to make it easier.